referrersdk/src/main/java/ir/cafebazaar/referrersdk/receiver/ReferrerReceiver.kt

```diff
@@ -0,0 +1,39 @@
+package ir.cafebazaar.referrersdk.receiver
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+
+internal class ReferrerReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context?, intent: Intent?) {
+        intent?.apply {
```

Comment:
Redundant apply, simply call `notifyObservers(intent)`

Reply:
I changed it to let function. It is needed to avoid notify observers with a null value.